### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by poudelprakash**

### CODE REVIEW

File: diamonds.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* Write a program which has an array of diamonds with different value e.g. [1,2,3,4,5]. 
+The program should distribute the diamonds equally to 3 people. For example: Person 1: 1,4 Person 2: 2,3 Person 3: 5 
+There may be any number of diamonds with any value. If diamonds cannot be equally distributed(e.g [1,2,3,4]), 
+the program should print message “Cannot distribute equally to 3 people”. */
+var diamonds = [2,4,5,3,1,6];
+var total=0;
+var i;
+var remove;
+var pn=1;
```
Comment (poudelprakash) [inline diamonds.js]: name properly, what is pn?

File: diamonds.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/* Write a program which has an array of diamonds with different value e.g. [1,2,3,4,5]. 
+The program should distribute the diamonds equally to 3 people. For example: Person 1: 1,4 Person 2: 2,3 Person 3: 5 
+There may be any number of diamonds with any value. If diamonds cannot be equally distributed(e.g [1,2,3,4]), 
+the program should print message “Cannot distribute equally to 3 people”. */
+var diamonds = [2,4,5,3,1,6];
+var total=0;
+var i;
+var remove;
```
Comment (poudelprakash) [inline diamonds.js]: what is remove? variable name should be meaningful

File: diamonds.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+var i;
+var remove;
+var pn=1;
+
+
+for(i=0; i<diamonds.length;i++){
+	total= total+diamonds[i];
+}
+
+if(total%3!=0){
```
Comment (poudelprakash) [inline diamonds.js]: extract number of people to variable, you should program in such a way that any change in requirements would need you minimal to no change

File: diamonds.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	console.log("Cannot distribute equally to 3 people");
+}
+else{
+	while(diamonds.length!=0){
+	for(i=0; i<diamonds.length;i++){
```
Comment (poudelprakash) [inline diamonds.js]: using `diamonds.length` like this evaluates length each iteration on loop, you can avoid this by extracting this in a variable.

File: diamonds.js (hunk 1 above)
Comment (anilpdl) [reply diamonds.js]: person number

Comment (poudelprakash) [reply diamonds.js]: so name it as personNumber, or currentPerson

**Pull request closed without merging**